src/pkg/cli/client/provider.go

```diff
@@ -120,6 +120,7 @@ type Provider interface {
 	DeleteConfig(context.Context, *defangv1.Secrets) error
 	Deploy(context.Context, *defangv1.DeployRequest) (*defangv1.DeployResponse, error)
 	Destroy(context.Context, *defangv1.DestroyRequest) (types.ETag, error)
+	DriverName() string
```

Comment:
terminology: let's stick to "provider" for this. I did use "driver" a year ago but it's for the low-level "start a task in a cloud" code.